docs/reference/troubleshooting/common-issues/hotspotting.asciidoc

```diff
@@ -259,8 +259,14 @@ direct indices:data/read/eql 10m           node_1  true
 // TEST[skip:illustrative response only]
 
 This surfaces a problematic <<eql-search-api,EQL query>>. We can gain 
-further insight on it via <<tasks,the task management API>>. Its response 
-contains a `description` that reports this query:
+further insight on it via <<tasks,the task management API>>,
+
+[source,console]
+----
+GET _tasks?pretty&human&detailed=true
```

Comment:
I'd recommend not using `pretty`, it can double the size of this already-pretty-heavyweight response on the wire. Also the `=true` is not required:

```suggestion
GET _tasks?human&detailed
```